packages/api/src/entities/User.ts

```diff
@@ -13,6 +13,9 @@ export class User extends Node<User> {
   @Property({ columnType: 'text' })
   githubId: string;
 
+  @Property({ columnType: 'text' })
+  githubUsername: string;
```

Comment:
Do we need this in addition to `githubId`?

Reply:
Yes, because the contribution query from graphql only returns the username.  You can't get the ID form graphql.  So this is how we know what contribution goes to what user.